Makefile

```
@@ -161,8 +161,8 @@ code.lobster-%:
 	lobster-python --out code.lobster lobster/tools/$(TOOL_PATH)
 
 unit-tests.lobster-%:
-	$(eval TOOL_PATH := $(subst -,/,$*))
-	lobster-python --activity --out unit-tests.lobster tests-unit/lobster-$(TOOL_PATH)
+	$(eval TOOL_NAME := $(subst _,-,$(notdir $(TOOL_PATH))))
```

Should it be `TOOL_NAME` instead of `TOOL_PATH` everywhere?
```suggestion
	$(eval TOOL_NAME := $(subst _,-,$(notdir $(TOOL_NAME))))
```

---

The TOOL_PATH variable contains the path of tools from `lobster/tools/core` folder as
core/ci_report
core/html_report
core/online_report
core/report
 
and it tries to find the tool in `tests-unit` folder as
 
core/lobster-ci_report
core/lobster-html_report
core/lobster-online_report
core/lobster-report
 
hence, this affects tracing reports, reports do not get generated.
 
So, I fetch the path from TOOL_PATH and then eliminate the "core/" part and replace "_" with "-"
 
The code gives the TOOL_NAME as
 
lobster-ci-report
lobster-html-report
lobster-online-report
lobster-report
 
and tracing reports then get generated successfully